tailwind.config.js

```diff
@@ -5,13 +5,21 @@ module.exports = {
     `./src/components/*.{js,jsx,ts,tsx}`,
     `./src/design-system/*.{js,jsx,ts,tsx}`,
     `./src/features/**/*.{js,jsx,ts,tsx}`,
+    `./src/views/**/*.{js,jsx,ts,tsx}`,
   ],
   darkMode: `class`,
   theme: {
     extend: {
       screens: {
         tn: `400px`,
       },
+      colors: {
```

Comment:
Remove these variables, use native builded in classes from tailwind palette like red-500, etc...

https://tailwindcss.com/docs/customizing-colors